### DISCUSSION TITLE
Deprecating Cocoa-pods.

### DISCUSSION THREAD

**Discussion by thomasMinshull** [kean/Nuke#525] [General]
We were about to integrate Nuke into our iOS project. Unfortunately, deprecating cocoa-pod support will likely put a hard stop to this. This project is a mixed RN and Swift project, and it's not yet possible to run SPM with RN, and even at the point when this becomes supported by RN it's doubtful that many RN modules would upgrade to support it. 

Mixing and matching dependency managers in a project is problematic because these tools are not aware of each other dependency graphs. 

Please reconsider deprecating cocoa-pods

**Comment by kean** [comment]
I know there are quite a few projects that still use CocoaPods or Carthage. So I'm not planning to remove the support for them any time soon. With deprecations, I just wanted to signal that it will eventually be removed. It doesn't mean there weren't be any more CocoaPods releases.

**Reply by thomasMinshull** [reply]
Great, that is good to know, and thank you for the quick reply.
